e2e/tests/accessibility.spec.ts

```diff
@@ -4,6 +4,10 @@ import { expect, test } from '../fixtures'
 // There are various known failures in the GOV.UK Design System:
 // https://github.com/alphagov/govuk-frontend/issues/1280#issuecomment-509588851
 const DISABLED_RULES: Record<string, Record<string, string[]>> = {
+  accordion: {
+    default: ['aria-allowed-attr'],
+    'with-summary-lines': ['aria-allowed-attr'],
```

Comment:
I would add a comment because the issue is not included in the known issues link https://github.com/alphagov/govuk-frontend/issues/1280#issuecomment-509588851

Reply:
It does have its own issue it seems https://github.com/alphagov/govuk-frontend/issues/2472